pkg/logging/logging.go

```diff
@@ -0,0 +1,107 @@
+// package logging is a small wrapper around github.com/k8snetworkplumbingwg/cni-log
+
+package logging
+
+import (
+	cnilog "github.com/k8snetworkplumbingwg/cni-log"
+)
+
+const (
+	labelCNIName     = "cniName"
+	labelContainerID = "containerID"
+	labelNetNS       = "netns"
+	labelIFName      = "ifname"
+	cniName          = "sriov-cni"
+)
+
+var (
+	logLevelDefault = cnilog.InfoLevel
+	containerID     = ""
+	netNS           = ""
+	ifName          = ""
+)
+
+// Init initializes logging with the requested parameters.
+func Init(logLevel, logFile, containerID, netns, ifName string) {
+	SetLogLevel(logLevel)
```

Comment:
Let's not export the setters anymore. They are only used in this package. Also, let's remove the setters for continaerid, netns and ifname.

Reply:
done